README.md

```diff
@@ -54,6 +64,74 @@ For a list of all configuration options for sumologicextension please refer to
 [sumologicextension]: ./pkg/extension/sumologicextension/
 [sumologicextension_configuration]: ./pkg/extension/sumologicextension#configuration
 
+## How to run
+
+```bash
+Usage:
+  otelcol-sumo [flags]
+
+Flags:
+      --add-instance-id             Flag to control the addition of 'service.instance.id' to the collector metrics. (default true)
+      --config string               Path to the config file
+  -h, --help                        help for otelcol-sumo
+      --log-format string           Format of logs to use (json, console) (default "console")
+      --log-level Level             Output level of logs (DEBUG, INFO, WARN, ERROR, DPANIC, PANIC, FATAL) (default info)
+      --log-profile string          Logging profile to use (dev, prod) (default "prod")
+      --mem-ballast-size-mib uint   Flag to specify size of memory (MiB) ballast to set. Ballast is not used when this is not specified. default settings: 0
+      --metrics-addr string         [address]:port for exposing collector telemetry. (default ":8888")
+      --metrics-level Level         Output level of telemetry metrics (none, basic, normal, detailed) (default basic)
+      --metrics-prefix string       Prefix to the metrics generated by the collector. (default "otelcol")
+      --set stringArray             Set arbitrary component config property. The component has to be defined in the config file and the flag has a higher precedence. Array config properties are overridden and maps are joined, note that only a single (first) array property can be set e.g. -set=processors.attributes.actions.key=some_key. Example --set=processors.batch.timeout=2s (default [])
+  -v, --version                     version for otelcol-sumo
+
+```
+
+### Binary
+
+Visit [releases page][releases_page] in order to download the binary for your OS
+and then run it as shown below:
+
+```bash
+$ curl -sLo otelcol-sumo https://github.com/SumoLogic/sumologic-otel-collector/releases/download/v0.0.5/otelcol-sumo-0.0.5-darwin_amd64
+$ chmod +x otelcol-sumo
+$ ./otelcol-sumo --version
+otelcol-sumo-darwin_amd64 version v0.0.5
+```
+
+[releases_page]: https://github.com/SumoLogic/sumologic-otel-collector/releases
+
+### Docker container
+
+Our container images are stored in AWS Public ECR under the following repository:
+`public.ecr.aws/sumologic/sumologic-otel-collector`.
+
+One can find all the available tags pushed to this repository at
+https://gallery.ecr.aws/sumologic/sumologic-otel-collector.
+
+In order to run Sumo Logic OT distro in a container one can use the following
+set of commands:
+
+```bash
+$ ls config.yaml
+config.yaml
+$ docker run --rm -ti --name sumologic-otel-collector -v $(pwd):/etc/ public.ecr.aws/sumologic/sumologic-otel-collector:0.0.4 --config /etc/config.yaml
```

Comment:
Could we explain why we mount full directory?

Reply:
I didn't think of mounting a single file, thanks! Done.

Reply:
Buuuut, what about state files in the future? :D

Reply:
Let's cover that in a separate PR :)